index.rb

```diff
@@ -0,0 +1,70 @@
+# frozen_string_literal: true
+
+require_relative 'lib/game'
+
+def render(board)
+  puts '   +----+----+----+----+----+----+----+----+'
+  board.positions.each_index do |row|
+    print "#{row}  "
+    board.positions[row].each_index do |column|
+      if board.positions[row][column].nil?
+        print '|    '
+      else
+        print "| #{board.positions[row][column]}  "
+      end
+    end
+    puts '|'
+    puts '   +----+----+----+----+----+----+----+----+'
+  end
+
+  print '   '
+  (0..7).each { |letter| print "  #{letter}  " }
+  puts ''
+end
+
+def input_figure(game)
+  puts 'Please enter the figure you want to move'
```

Comment:
This is an awful user experience, but most of all it has a bug. What if multiple of my figures have access to the same target field? It's going to pick one for me, and that is unacceptable.

The UX here should demand I provide a move identifier like "a2 to b4"